dangerfile.js

```diff
@@ -0,0 +1,49 @@
+import { message, danger, warn, markdown } from "danger";
+const prettier = require("prettier");
+const fs = require("fs");
+let pr = danger.github.pr;
+if (pr.body.length === 0) {
+  fail("Please add a description to your PR.");
+}
+
+var path = require("path");
+if (pr.author_association == "FIRST_TIME_CONTRIBUTOR") {
+  message(
+    "Since this is your first contribution, here are some links while you wait for a review.\n[Our contributing guidelines](https://github.com/ScratchAddons/ScratchAddons/blob/master/CONTRIBUTING.md)"
```

Comment:
You should go with a friendly tone like "hello, this is your first contribution, blablabla". I'm actually going to make this later with https://probot.github.io/apps/welcome/, which also includes welcoming on their first PR and issue, but this could be it for now.

https://github.com/ScratchAddons/ScratchAddons/blob/master/CONTRIBUTING.md is quite dead, let's remove it for now?